src/java/org/apache/ivy/util/url/HttpClientHandler.java

```diff
@@ -177,7 +177,7 @@ public URLInfo getURLInfo(URL url, int timeout) {
         return UNAVAILABLE;
     }
 
-    private boolean checkStatusCode(URL url, HttpMethodBase method) throws IOException {
+    private boolean checkStatusCode(URL url, HttpMethodBase method) {
```

Comment:
Similar comment as here https://github.com/apache/ant-ivy/pull/52/files#r127395633 applies to this line